### PR TITLE
Ci add federated integ test

### PR DESCRIPTION
**Pull request by crasu**
Test PR for planning around with the feg CI setup.

This PR is a clone of #12633. 

The feg integ tests now work on this branch. However #12633 still needs some polishing before it can  be merged into master.

### CODE REVIEW

**Review by uri200 — COMMENTED** [review]
lets separate those changes into groups.

I woukd advise to put hss changes on a different PR and work from there

**Review by uri200 — COMMENTED** [review]
btw good catch with the hss issue!

File: lte/gateway/fabfile.py [hunk 1]
```diff
@@ -219,23 +220,86 @@ def s1ap_setup_cloud():
...
+
         local("fab configure_orc8r")
-        local("fab test_connectivity")
+
+        #FEG may need to be restarted to reload some configs
+        print("Wait for configuration to be available")
+        sleep(60)
+        print("Restart hss")
+        local('fab restart_feg_hss')
+        sleep(20)
```
Comment (uri200) [inline lte/gateway/fabfile.py]: can you please check if the test works without those lines (296 to 300) ?

Your patch may have fixed the need for that hack

File: .github/workflows/federated-integ-test.yml [hunk 2]
```diff
@@ -0,0 +1,220 @@
+---
+
+name: Federated integ test
+
+on:  # yamllint disable-line rule:truthy
```
Comment (Neudrino) [inline .github/workflows/federated-integ-test.yml:5]: With https://github.com/magma/magma/pull/12990 merged, we can actually remove the lint ignore.

**Pull request closed without merging**